test/helpers/kubectl.go

```diff
@@ -1842,6 +1873,78 @@ func (kub *Kubectl) CiliumServicePreFlightCheck() error {
 	return nil
 }
 
+func (kub *Kubectl) DeployETCDOperator() error {
+	const etcdOperatorPath = "../examples/kubernetes/addons/etcd-operator/"
```

Comment:
Would be helpful in Deploy / Delete functions to add logs or invocations of `By` to describe what's going on. It will make it easier to debug any potential issues in the future.